features/result.feature

```diff
@@ -14,473 +14,480 @@ Background:
+ And I fill in "MED" in "div.form-group:nth-child(1) > input:nth-child(2)" field
+ And I click "Add" button
+ And I click "fa-briefcase" icon
  And I click to OK on confirm dialog
  And I am on Task archive page
- And I click on "dropdownMenu1" id button
+ And I click button with id "dropdownMenu1"
  And I click "View" link within dropdown menu
- Then I click on ".file-download-link" sign
+ Then I click element with css ".file-download-link"
  #no notification can be found
 
 @javascript
 Scenario: Delete archived Text result
  Given I am on Task results page
- And I click on "Text" button
+ And I click "Text" button
  And I fill in "LJ ZOO" in "#result_name" field
  And I fill in "Živali pa so se odpravile dalje po svetu." in "#result_text_attributes_textarea" rich text editor field
- And I click on "save-result" class button
- And I click on "form-submit-link" class button
+ And I click "Add" button
+ And I click "fa-briefcase" icon
  And I click to OK on confirm dialog
  And I am on Task archive page
- And I click on "dropdownMenu1" id button
+ And I click button with id "dropdownMenu1"
  And I click "Delete" link within dropdown menu
  And I click to OK on confirm dialog
  Then I should see "Sucessfully removed result LJ ZOO from task Experiment design." flash message
 
 @javascript
 Scenario: Delete archived Table result
  Given I am on Task results page
- And I click on "Table" button
- And I click on "save-result" class button
- And I click on "form-submit-link" class button
+ And I click "Table" button
+ And I click "Add" button
+ And I click "fa-briefcase" icon
  And I click to OK on confirm dialog
  And I am on Task archive page
- And I click on "dropdownMenu1" id button
+ And I click button with id "dropdownMenu1"
  Then I click "Delete" link within dropdown menu
  And I click to OK on confirm dialog
  Then I should see "Sucessfully removed result from task Experiment design." flash message
 
 @javascript
 Scenario: Delete archived File result
  Given I am on Task results page
- And I click on "File" button
+ And I click "File" button
  And I attach file "Moon.png" to the drag-n-drop field
- And I fill in "MED" in "div.form-group:nth-child(1) > input:nth-child(2)" field  
- And I click on "save-result" class button
- And I click on "form-submit-link" class button
+ And I fill in "MED" in "div.form-group:nth-child(1) > input:nth-child(2)" field
+ And I click "Add" button
+ And I click "fa-briefcase" icon
  And I click to OK on confirm dialog
  And I am on Task archive page
- And I click on "dropdownMenu1" id button
+ And I click button with id "dropdownMenu1"
  And I click "Delete" link within dropdown menu
  And I click to OK on confirm dialog
  Then I should see "Sucessfully removed result MED from task Experiment design." flash message
+
+ # We need to optimize some tests... When you deleting archived result, we always adding result. This should be initial
+ # state there, already
+
+
+ #### JUST TESTING, will be removed
```

Comment:
When it will be removed?

Reply:
Samo will remove it. It's there for demonstration.

Reply:
OK